src/test/java/teammates/test/cases/action/StudentFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -653,6 +653,9 @@ public void testExecuteAndPostProcess() throws Exception {
         ______TS("Successful case: const sum: typical case");
 
         fq = fqDb.getFeedbackQuestion("CONSTSUM Session", "FSQTT.idOfTypicalCourse1", 1);
+        FeedbackQuestionAttributes fq2 = fqDb.getFeedbackQuestion(
```

Comment:
What are these changes for?

Reply:
Add a comment, say it is const sum question with only min constraint.

BTW. You don't need to create a new JSON data (and u already know that is very troublesome and complicated). You can use logic DB to modify the min/max constraint.  

In addition, if u are testing question with only min constraint, how about question with only max constraint?